src/components/Routes.js

```diff
@@ -30,6 +31,7 @@ const Routes = () => (
     <Route exact path="/login" component={Login} />
     <Route exact path="/articles/create" component={CreateArticles} />
     <Route exact path="/privacy" component={Privacy} />
+    <Route exact path="/article/:slug" component={ViewArticle} />
```

Comment:
How would you like using articles in the plural for the path, as the slug now specifies the article from the articles? Good convention i hope

Reply:
Nice suggestion mike. But we will have a conflict since we already have routes such as articles/create, articles/edit. remember the one I used requires a parameter thus it may take edit and create as parameters yet they are not